pkg/test/framework/api/descriptors/descriptors.go

```diff
@@ -116,6 +116,15 @@ var (
 		},
 	}
 
+	HealthCheckApps = component.Descriptor{
```

Comment:
This looks like it is out of date, don't need this anymore right?

Reply:
Removed.